setup.cfg

```diff
@@ -0,0 +1,10 @@
+[pycodestyle]
+max-line-length = 119
```

Comment:
Oh, one another question: why 119?

Reply:
Most people find 80 too short. PEP actually specifies 79 as a max so 119 is a more reasonable equivalent.

Reply:
Contribution guideline still mentions PEP8.

Reply:
Until now we've not had an easy way to consciously ignore certain warnings (and specifically in certain places as appropriate). Now that we do, I don't think it's unreasonable to consider 120 characters given the size of modern screens and post-pep8 features like lint markers, type hinting, etc. Perhaps if someone with poor vision wanted to split their screen vertically it would be a problem for them. PEP8 is supposed to make the code exceptionally readable despite a simple text editor. Readability is so greatly improved for an IDE, even my straightforward vim setup.

Maybe we can @all give a thumbs up to this comment if we're happy with 120 character lines and a thumbs down if we want to stick to the letter rather of PEP8?

Can we get a decision on this please, @PeterSurda? There will be much more time-consuming line-chomping and risk enforcing 80 characters. I'm happy to do it but maybe that's not priority 1, 2 or 3 right now?

Reply:
Now what you do? O_o Best way to ignore certain warnings is to not use any lints: neither codacy no flake8 in editor. Then the code will slowly become unreadable and unpredictable.

Readability is not about screen size.

Reply:
If 80 isn't practical, 119 is ok.